doc/enterprise/audit_log.rst

```diff
@@ -647,8 +619,8 @@ How many events can be recorded?
 
 If you write to a file, the size of the Tarantool audit module is limited by the disk space.
 If you write to a system logger, the size of the Tarantool audit module is limited by the system logger.
-If you write to a pipe, the size of the Tarantool audit module is limited by the system buffer
-if the ``audit_nonblock`` = ``false``; if ``audit_nonblock`` = ``true``, there is no limit.
+If you write to a pipe, the size of the Tarantool audit module is limited by the system buffer.
+If the ``audit_log.nonblock = false``, if ``audit_log.nonblock`` = ``true``, there is no limit.
 However, it is not recommended to use the entire memory, as this may cause performance degradation
 and even loss of some logs.
```

Comment:
That's an unclear recommendation, because there are no configurable memory limit for the audit log.